website/static/js/changeMessage.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+var ko = require('knockout');
+var oop = require('js/oop');
+/** Change the flashed status message */
+
+var ChangeMessage = oop.defclass({
```

Comment:
Naming classes as verbs is ambiguous. Maybe rename to `ChangeMessageMixin` to indicate that this is meant to be subclassed.  The docstring could use clarification as well. Perhaps: 

```js
/**
 * ViewModel mixin for displaying form input help messages.
 * Adds message and messageClass observables that can be changed with the
 * changeMessage method.
 */
```